ibc/chain.go

```diff
@@ -115,6 +115,8 @@ type RollApp interface {
 	ShowSequencer(ctx context.Context) (string, error)
 	// Get Sequencer
 	GetSequencer() string
+	// Get Validator Genesis Dir
+	GetHomeDir() string
```

Comment:
We already have HomeDir on CosmosChain, you can use it.

Reply:
good catch